account_operating_unit/views/account_move_view.xml

```diff
@@ -90,26 +89,28 @@
                     'operating_unit_id': operating_unit_id}
                 </attribute>
             </field>
-            <xpath
-                expr="//field[@name='invoice_line_ids']/tree/field[@name='analytic_account_id']"
-                position="attributes"
-            >
-                <attribute name="domain">
-                    ['|', ('company_id', '=', False), ('company_id', '=',
-                    parent.company_id), '|', ('operating_unit_ids', '=',
-                    context.get('operating_unit_id', False)), ('operating_unit_ids',
-                    '=', False)]
-                </attribute>
-            </xpath>
-            <xpath
-                expr="//field[@name='invoice_line_ids']/form/sheet/group/field[@name='analytic_account_id']"
-                position="attributes"
-            >
-                <attribute name="domain">
-                    ['|', ('operating_unit_ids', '=', context.get('operating_unit_id',
-                    False)), ('operating_unit_ids', '=', False)]
-                </attribute>
-            </xpath>
+<!--            FIXME: this needs to be replaced in the js domain in widget="analytic_distribution" in
```

Comment:
It would be nice to keep this feature, let's see if someone can inherit from that function.